src/PowerShellEditorServices/Services/Symbols/ReferenceTable.cs

```diff
@@ -79,6 +79,7 @@ internal void EnsureInitialized()
         }
 
         _parent.ScriptAst.Visit(new SymbolVisitor(_parent, AddReference));
+        RegionVisitor.GetRegionsInDocument(_parent, AddReference);
```

Comment:
@fflaten I'm contemplating a way of moving this out of the reference table. Like ideally I want to call `GetRegionsInDocument` _only_ in `DocumentSymbolHandler` and get the region symbols separately. This would also simplify the rest of the PR because we wouldn't have to special case excluding them.